luzer/tests/test_unit.lua

```diff
@@ -145,6 +147,9 @@ ok, err = pcall(fdp.consume_integer)
 assert(ok == false)
 assert(err ~= nil)
 
+local i = fdp:consume_integer(1, MAX_INT)
```

Comment:
It is better to use `MAX_INT, MAX_INT` to check both min and max values. And check the returned value below to be equal.

Reply:
Added:

```lua
local max_int = fdp:consume_integer(MAX_INT, MAX_INT)
assert(type(max_int) == "number")
assert(max_int == MAX_INT)

local min_int = fdp:consume_integer(MIN_INT, MIN_INT)
assert(type(min_int) == "number")
assert(min_int == MIN_INT)

local i = fdp:consume_integer(MIN_INT, MAX_INT)
assert(type(i) == "number")
```